### PR TITLE
internal scrollbars removed for attendees list

### PR DESCRIPTION
**Pull request by poush**
Fixes thing mention in #3138

### CODE REVIEW

File: app/templates/gentelella/users/events/tickets/attendees.html [hunk 1]
```diff
@@ -154,19 +154,16 @@
 
     <h3>{{ _("View Attendees") }}</h3>
     <br>
-    <table class="table with-datatable" id="orders-table">
+    <table class="table with-datatable" id="orders-table" width="100%">
         <thead>
         <tr>
-            <th>
+            <th width="20px">
```
Comment (niranjan94) [inline app/templates/gentelella/users/events/tickets/attendees.html]: Please specify the column widths via datatables configuration only

File: app/templates/gentelella/users/events/tickets/attendees.html [hunk 2]
```diff
@@ -253,18 +248,18 @@
     <script src="{{ url_for('static', filename='js/events/ticket_sales/attendees.js')}}"></script>
 
     <script type="text/javascript">
-        var options = ['#completed', '#placed', '#pending', '#expired', '#cancelled', '#all'];  
+        var options = ['#completed', '#placed', '#pending', '#expired', '#cancelled', '#all']; 
         $.fn.dataTable.ext.search.push(
             function (settings, data, dataIndex) {
                 var user_option = $("input[name=show_state]:checked").val();
-                var state = data[2].trim() || 'pending';
+                var state = data[1] || 'pending';
```
Comment (SaptakS) [inline app/templates/gentelella/users/events/tickets/attendees.html]: Please add `.trim()` function to ensure that it's not just whitespaces. Thanks.

File: app/templates/gentelella/users/events/tickets/attendees.html (hunk 1 above)
Comment (poush) [inline app/templates/gentelella/users/events/tickets/attendees.html]: @niranjan94 Updated

File: app/templates/gentelella/users/events/tickets/attendees.html (hunk 2 above)
Comment (poush) [inline app/templates/gentelella/users/events/tickets/attendees.html]: @SaptakS added

**Comment by poush** [comment]
screen shot 2017-02-18 at 11 48 34 pm

**Pull request merged**